B827EBFFFE937A49.json

```
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "eui-b827ebfffe937a49", 
```

Please use the upper cased EUI, without the `eui-` prefix, i.e.:

```suggestion
		"gateway_ID": "B827EBFFFE937A49", 
```